backend/routes/loginAuth.ts

```diff
@@ -11,6 +11,12 @@ loginAuthRouter.get('/password',
     res.render('login');
   });
 
+loginAuthRouter.post("/password", passport.authenticate('local'),
+  function (req, res) {
+    console.log('user from local auth: ' + (req.user as User).id);
```

Comment:
as a side note we generally want to use `console.info` for stuff that you want to log that is relevant to log (aka, not just for debugging)

some background reading: https://sematext.com/blog/logging-levels/